src/spatialdata_plot/pl/basic.py

```diff
@@ -319,6 +321,8 @@ def render_images(
             Color to be used for NAs values, if present.
         alpha
             Alpha value for the shapes.
+        quantile_norm
```

Comment:
which quantile?

Reply:
Found in the private function, I would maybe expose the quantiles to the user via this function

Reply:
It's based on https://github.com/CSBDeep/CSBDeep/blob/deb54947a734b959681ead6bde24c6143805663f/csbdeep/utils/utils.py#L51, don't know their exact reason behind it though. I could just link this function?

Reply:
> Found in the private function, I would maybe expose the quantiles to the user via this function

Good idea!

Reply:
I would like the function yes.